packages/core/src/NodeExecuteFunctions.ts

```diff
@@ -2674,7 +2674,7 @@ const getFileSystemHelperFunctions = (node: INode): FileSystemHelperFunctions =>
 	},
 
 	getStoragePath() {
-		return path.join(getUserN8nFolderPath(), `storage/${node.type}`);
+		return path.join(Container.get(InstanceSettings).n8nFolder, `storage/${node.type}`);
```

Comment:
We could move this to `instanceSettings`.

Reply:
for now, I'd like to leave this. I added `getStoragePath` to be able to store `.whl` files for python code nodes. but, I might want to change this to not have a generic storage path for all nodes.